lib/mqtt/MqttClient.js

```diff
@@ -52,6 +52,9 @@ class MqttClient {
             }
         });
 
+        Logger.onLogMessage((line) => {
```

Comment:
I really like the .onLogMessage, however I wasn't really expecting that logging to mqtt would be implemented straight away. Sorry for being unclear there

Due to issues like secrets leaking somewhere, I'd propose commenting out the mqtt part for now